app/_data/mesh_rbac_2.11.x.yaml

```diff
@@ -0,0 +1,333 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
how is this retrieved? How is this retrieved for kong-mesh? Can't we just have a command line to do this directly from helm?

Reply:
Yes, it's done:
- https://github.com/kumahq/kuma/pull/13581
- https://github.com/kumahq/kuma-website/pull/2308